CHANGELOG.md

```diff
@@ -74,6 +75,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Bindings for Super/Command + F1-F12
 - Automatic config generation
+- Deprecated `scrolling.faux_multiplier`, use `scrolling.multiplier` instead
```

Comment:
We should probably point out that the new escape can be used to enable/disable faux scrolling completely. We might need to break this over two lines with all the info necessary, but that's not the end of the world.